test/remoting.integration.js

```diff
@@ -6,6 +6,12 @@ var app = require(path.join(SIMPLE_APP, 'server/server.js'));
 var assert = require('assert');
 
 describe('remoting - integration', function() {
+  before(function(done) {
+    if (app.booting) {
+      return app.once('booted', done);
+    }
+    done();
+  });
```

Comment:
I would simply move it into the `beforeEach` hook code here: https://github.com/strongloop/loopback/blob/0ad150cb6ee1501d5bcbdb077ee5621cdb64c407/test/helpers/loopback-testing-helper.js#L21-L28